x-aeson/src/X/Data/Aeson.hs

```diff
@@ -19,6 +19,7 @@ import           Data.Text
 import           Data.Text.Encoding (decodeUtf8, encodeUtf8)
 import           Data.ByteString.Lazy (toStrict)
 
+import           Prelude -- AMP compatibility workaround
```

Comment:
Redundant Applicative warnings. This is the usual migration trick, but given we maintain our own Prelude, maybe not so good. Can be replaced with `NoImplicitPrelude` and explicit imports `Control.Monad` `Data.Either`, `Data.Maybe`, `GHC.Base`